utilities/cluster_config_utils.py

```diff
@@ -159,6 +159,11 @@ def get_revs_limit(cluster_config):
     return cluster["environment"]["revs_limit"]
 
 
+def get_logging(cluster_config):
```

Comment:
This can be renamed to reflect the purpose like get_log_redaction_level

Reply:
done